### DISCUSSION TITLE
"Create your first list" breaks mobile layout

### DISCUSSION THREAD

**Discussion by yoursunny** [community/community#8660] [Other Feature Feedback, Questions, & Ideas] [answered]
I have [stars - sort by recently active](https://github.com/yoursunny?direction=desc&sort=updated&tab=stars) page bookmarked to keep track of my favorite projects.
It used to render correctly on mobile, as in the content fits the screen width.

This week, "Start your first list" suggestion showed up, and it seems to break mobile layout.
The content appears wider at the "sort by" dropdown widget than what it used to be, and the whole page is zoomed out, making it harder to see the repositories.

![chrome_screenshot_1639139853745](https://user-images.githubusercontent.com/1141273/145575637-e3ca7ed0-afe0-417b-b9f7-0fe3c1354223.png)

**Comment by yoursunny** [comment]
This page was OK before the "Create your first list" showed up.
There was horizontal scrolling on the three dropdown widgets, but the rest of the page fits the width.

**Comment by kadirdemirel** [comment]
Really cool feature :)

**Comment by yoursunny** [comment, marked as answer]
This issue is now fixed.
